### DISCUSSION TITLE
Request a problem solution

### DISCUSSION THREAD

**Discussion by chaaibz** [ant-media/Ant-Media-Server#6509] [General]
I have trouble knowing username and passworld 
When you connect to my server, knowing that when you enter the server, a direct page pops up username and passwold Having difficulty getting over it

**Comment by amarantmeida** [comment]
@chaaibz May I know on which cloud platform you have hosted your Ant Media Server and did you host Enterprise Edition or Community Edition?

**Reply by chaaibz** [reply]
Community Edition

**Comment by chaaibz** [comment]
Community Edition
________________________________
De : Amarnath Thodupunoori ***@***.***>
Envoyé : vendredi 19 juillet 2024 13:42
À : ant-media/Ant-Media-Server ***@***.***>
Cc : chaaibz ***@***.***>; Mention ***@***.***>
Objet : Re: [ant-media/Ant-Media-Server] Request a problem solution (Discussion #6509)


@chaaibz<https://github.com/chaaibz> May I know on which cloud platform you have hosted your Ant Media Server and did you host Enterprise Edition or Community Edition?

**Reply by amarantmeida** [reply]
Hi @chaaibz 
Can you please confirm on which cloud platform you have hosted your Ant Media server.

**Reply by Mohit-3196** [reply]
I think it is Linode @amarantmeida

**Reply by amarantmeida** [reply]
@chaaibz If you have launched a new server, then you may have set the root password during the launch.
​
If not, then first you have to stop the server and set the root password. Check out the below link to reset the root password.
https://www.linode.com/docs/products/databases/managed-databases/guides/reset-root-password/
​
After this, please raise a ticket with us using this link-  https://antmedia.io/ contact option. Once your root password is reset, please share your server's public IP and root password so I will reset the username and password for you.
